finalization_state_restoration is flaky

finalization_state_restoration is a floating test and flakes out every now and then.

Full log: [finalization_state_restoration.txt](https://github.com/dtr-org/unit-e/files/3060960/finalization_state_restoration.txt)

Failure:

```
[1mfinalization_state_restoration.py[0m failed, Duration: 154 s

[1mstdout:
[0m2019-04-09 20:17:26.720000 TestFramework (INFO): Initializing test directory /tmp/unite_test_runner_20190409_200057/finalization_state_restoration_652
2019-04-09 20:17:26.720000 TestFramework (INFO): Debug file at /tmp/unite_test_runner_20190409_200057/finalization_state_restoration_652/node0/regtest/debug.log
Starting node 0 with args: -esperanzaconfig={"epochLength": 5, "minDepositSize": 1500}
Starting node 1 with args: -esperanzaconfig={"epochLength": 5, "minDepositSize": 1500} -validating=1
2019-04-09 20:17:26.720000 TestFramework.node0 (INFO): Running command: /home/travis/build/dtr-org/unit-e/build/unit-e-i686-pc-linux-gnu/src/unit-e -datadir=/tmp/unite_test_runner_20190409_200057/finalization_state_restoration_652/node0 -server -keypool=1 -discover=0 -rest -logtimemicros -debug -debugexclude=libevent -debugexclude=leveldb -mocktime=0 -uacomment=testnode0 -esperanzaconfig={"epochLength": 5, "minDepositSize": 1500}
2019-04-09 20:17:26.723000 TestFramework.node1 (INFO): Running command: /home/travis/build/dtr-org/unit-e/build/unit-e-i686-pc-linux-gnu/src/unit-e -datadir=/tmp/unite_test_runner_20190409_200057/finalization_state_restoration_652/node1 -server -keypool=1 -discover=0 -rest -logtimemicros -debug -debugexclude=libevent -debugexclude=leveldb -mocktime=0 -uacomment=testnode1 -esperanzaconfig={"epochLength": 5, "minDepositSize": 1500} -validating=1
2019-04-09 20:17:27.521000 TestFramework (INFO): Setup deposit
2019-04-09 20:17:29.127000 TestFramework (INFO): Generate few epochs
2019-04-09 20:20:00.555000 TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/home/travis/build/dtr-org/unit-e/build/unit-e-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 156, in main
    self.run_test()
  File "/home/travis/build/dtr-org/unit-e/build/unit-e-i686-pc-linux-gnu/test/functional/finalization_state_restoration.py", line 67, in run_test
    sync_mempools([p, v])
  File "/home/travis/build/dtr-org/unit-e/build/unit-e-i686-pc-linux-gnu/test/functional/test_framework/util.py", line 579, in sync_mempools
    ["\nNode %d: %s" % entry for entry in mempools.items()]))
AssertionError: Mempool sync failed:
Node 0: set()
Node 1: {'ca19fb1ba91394a8b3d671911b720de65f465411d13bf340375405767ee42d47'}
2019-04-09 20:20:00.556000 TestFramework (INFO): Stopping nodes
2019-04-09 20:20:00.664000 TestFramework (WARNING): Not cleaning up dir /tmp/unite_test_runner_20190409_200057/finalization_state_restoration_652
2019-04-09 20:20:00.664000 TestFramework (ERROR): Test failed. Test logging available at /tmp/unite_test_runner_20190409_200057/finalization_state_restoration_652/test_framework.log
2019-04-09 20:20:00.664000 TestFramework (ERROR): Hint: Call /home/travis/build/dtr-org/unit-e/build/unit-e-i686-pc-linux-gnu/test/functional/combine_logs.py '/tmp/unite_test_runner_20190409_200057/finalization_state_restoration_652' to consolidate all logs
```

--- a/test/functional/finalization_state_restoration.py
+++ b/test/functional/finalization_state_restoration.py
@@ -88,7 +88,7 @@ class FinalizatoinStateRestoration(UnitETestFramework):
                                      'lastFinalizedEpoch': 5,
                                      'validators': 1})
 
-        # connect validator and chek how it votes
+        # connect validator and check how it votes
         self.wait_for_vote_and_disconnect(v, p)
         generate_block(p, count=1)
 

